panic: runtime error: invalid memory address or nil pointer dereference

Hello Team,

I'm getting below error when I tried to run aztfy command to import all resources inside a resource group. this resource group has around 500 resources inside it.

   Azure Terrafy

panic: runtime error: invalid memory address or nil pointer dereference
                                                                       [signal 0xc0000005 code=0x0 addr=0x0 pc=0x1660809]

                                                                                                                         goroutine 176 [running]:
                                                                                                                                                 github.com/magodo/azlist/azlist.listDirectChildResource({0x2c4b110, 0xc000038100}, 0xc003199780, 0x10000c0032c7290?, {{0x2c4f818?, 0xc003458cd0?}, 0xc00273bce0?})
                                                                        github.com/magodo/azlist@v0.0.0-20220923015326-17b16cc9b805/azlist/azlist.go:347 +0x129
                                                                                                                                                               github.com/magodo/azlist/azlist.ListChildResource.func2()
                                                                                                                                                                                                                                github.com/magodo/azlist@v0.0.0-20220923015326-17b16cc9b805/azlist/azlist.go:285 +0x47
                                                                         github.com/magodo/workerpool.(*workPool).Run.func1(0x0?)
                                                                                                                                        github.com/magodo/workerpool@v0.0.0-20211124060943-1c48f3e5a514/workerpool.go:75 +0x99
                                                                                                                                                                                                                              created by github.com/magodo/workerpool.(*workPool).Run
                                                github.com/magodo/workerpool@v0.0.0-20211124060943-1c48f3e5a514/workerpool.go:66 +0x5d

--- a/azlist/azlist.go
+++ b/azlist/azlist.go
@@ -344,7 +344,12 @@ func listDirectChildResource(ctx context.Context, client *Client, schemaTree ARM
 		})
 	}
 
-	for crt, entry := range schemaTree[rt].Children {
+	schemaEntry := schemaTree[rt]
+	if schemaEntry == nil {
+		return result
+	}
+
+	for crt, entry := range schemaEntry.Children {
 		version := entry.Versions[len(entry.Versions)-1]
 		pager := client.resource.NewListChildPager(pid.String(), crt, version)
 		for pager.More() {

